Tensile/SolutionStructs.py

```diff
@@ -50,6 +50,16 @@ def reject(state, *args):
     for a in args:
       print(a)
     #traceback.print_stack(None, 2)
+    if globalParameters["_DebugCreateLibrary_Internal"]:
```

Comment:
PrintSolutionRejectionReason is false in TensileCreateLibrary?
if so, TensileCreateLibrary will not enter this condition.

Reply:
Default is false since we don't want to print the bunch of msgs when rocBlas build or doing benchmarks. So you need to add the --global-parameters=PrintSolutionRejectionReason=True when we are debugging purely via TensileCreateLibrary

Reply:
IMHO we can do away this if-clause by simply printing the message whenever the keys ("SolutionIndex", "SolutionNameMin", ...) are available. So we can avoid adding an extra parameter

Reply:
@rosenrodt Sounds good, I was also wondering how to avoid this "_DebugCreateLibrary_Internal" from being overrideen by Config.